fdbclient/NativeAPI.actor.cpp

```diff
@@ -3443,8 +3463,9 @@ ACTOR Future<Key> getKey(Reference<TransactionState> trState,
 	}
 }
 
-ACTOR Future<Version> waitForCommittedVersion(Database cx, Version version, SpanID spanContext) {
-	state Span span("NAPI:waitForCommittedVersion"_loc, { spanContext });
+ACTOR Future<Version> waitForCommittedVersion(Database cx, Version version, SpanContext spanContext) {
+	// TODO - ljoswiak, Parent or link? Below is linked constructor with empty parent.
```

Comment:
I think we resolved the above offline (decided on parent I believe). Remove todo